data.json

```diff
@@ -83,8 +83,9 @@
     {
       "tag": "fontmetrics",
       "name": "Font Metrics API",
+      "spec": "https://drafts.css-houdini.org/font-metrics-api/",
       "explainer": "https://github.com/w3c/css-houdini-drafts/blob/master/font-metrics-api/README.md",
-      "status": "none"
+      "status": "fpwd"
```

Comment:
The status of document is "A Collection of Interesting Ideas". It's not a First Public Working Draft yet

Reply:
I should have checked more thoroughly. Thanks for catching that